dnf/cli/commands/repoquery.py

```diff
@@ -475,6 +475,19 @@ def run(self):
                         rels = getattr(pkg, OPTS_MAPPING[self.opts.packageatr])
                     for rel in rels:
                         pkgs.add(str(rel))
+            if self.opts.resolve:
```

Comment:
Please in case --resolve try to use rel without conversion into str, but using set still sounds good.
Additionally please rise exception in configure if resolve is not used with one of packageatr option

Reply:
If we want to take advantage of putting rel to set, reldep object has to be hashable. Patch is here:
https://github.com/rpm-software-management/libdnf/pull/440